packages/cdk/apps/context/cdk.json

```diff
@@ -1,3 +1,6 @@
 {
-  "app": "npx -p typescript -p ts-node ts-node --prefer-ts-exts app.ts"
+  "app": "npx -p typescript -p ts-node ts-node --prefer-ts-exts app.ts",
+  "context": {
+    "@aws-cdk/core:bootstrapQualifier": "agc"
```

Comment:
Just to clarify, what is the behavior when customers upgrades from the existing AGC release to this one? Are our customers able to update their stacks or do they need to manually destroy them all as the assets references don't exist anymore and then deploy the contexts again (i.e. The context stack is currently using stored variables in cdk-bootstrap but now when we update the stack it is looking for the variables in agc-cdk-bootstrap which during an update don't exist yet)?

Reply:
What is the behavior/expectation today when upgrading? They might need to destroy their contexts and reactivate their account, but I've also done that every-time I've upgraded as well.

Reply:
The current expected behavior I believe is just to re-deploy the contexts/account